.github/workflows/daily-smoke-tests.yml

```diff
@@ -12,3 +12,17 @@ jobs:
     uses: ./.github/workflows/run-tests.yml
     with:
       node-matrix: "[{version: 'lts/*', artifact: 'lts'}, {version: 'latest', artifact: 'latest'}]"
+  v4-smoke-test:
```

Comment:
Added this because there's probably value in continuing to run the v4 smoke tests on a daily basis until v4 gets End-Of-Life'd. Prepared to be talked out of that, though.